caffe2/quantization/server/fully_connected_dnnlowp_op.h

```diff
@@ -62,4 +68,884 @@ class FullyConnectedDNNLowPOp
   std::int32_t in_qparams0_zero_point_old_ = 0;
 }; // class FullyConnectedDNNLowPOp
 
+using namespace std;
+
+template <typename T, bool ReluFused>
```

Comment:
Can we also do explicit instantiation? We want to minimize things we put in header files.